resolwe_bio/tools/bed_file_corrections_genome_browsers.py

```diff
@@ -9,17 +9,27 @@
 import argparse
 import pandas as pd
 
+from pandas.errors import EmptyDataError
+from resolwe_runtime_utils import error
+
 parser = argparse.ArgumentParser(description=__doc__)
 parser.add_argument('-f', '--bed_file', help="Bed file.")
 args = parser.parse_args()
 
-df = pd.read_csv(args.bed_file, delimiter='\t', header=None, dtype=str)
-df.iloc[:, 4] = pd.to_numeric(df.iloc[:, 4]).round().astype(int)
-df.iloc[:, 4] = df.iloc[:, 4].clip(upper=1000)
+try:
+    df = pd.read_csv(args.bed_file, delimiter='\t', header=None, dtype=str)
+except EmptyDataError:
+    print(error(
+        f'The input BED file {args.bed_file} is empty. Your analysis might '
+        f'have failed to identify regions of interest (peaks, junctions, etc.)'
+    ))
+else:
```

Comment:
I don't think you need an else clause here. Just checking for an empty file (try/catch) should be enough to trip the script and exit or process if DataFrame contains any data.

Reply:
It's not necessary to have the `else`, but it might be a better design choice. I did some reading on this topic, and found a good explanation here: https://stackoverflow.com/questions/3996329/else-clause-in-try-statement-what-is-it-good-for/3996378#3996378

Reply:
I think these two cases are equivalent and is a matter of personal preference.